src/ImageSharp/Metadata/Profiles/Exif/ExifReader.cs

```diff
@@ -20,7 +19,7 @@ public ExifReader(byte[] exifData)
     {
     }
 
-    public ExifReader(byte[] exifData, MemoryAllocator allocator)
+    public ExifReader(byte[] exifData, MemoryAllocator? allocator)
         : base(new MemoryStream(exifData ?? throw new ArgumentNullException(nameof(exifData))), allocator)
```

Comment:
It looks like you've changed the wrong field here. `allocator` is not null `exifData` is maybe null

Reply:
Oh! I misread. We NEVER call a version of the reader that passes an allocator.